src/test/java/org/openstreetmap/atlas/utilities/configuration/MergedConfigurationTest.java

```diff
@@ -18,33 +18,61 @@
  *
  * @author brian_l_davis
  * @author jklamer
+ * @author cameron_frenette
  */
 public class MergedConfigurationTest
 {
+    private static final String BASE_CONFIGURATION_JSON = "org/openstreetmap/atlas/utilities/configuration/application.json";
+    private static final String BASE_CONFIGURATION_YAML = "org/openstreetmap/atlas/utilities/configuration/application.json";
 
-    private static final String BASE_CONFIGURATION = "org/openstreetmap/atlas/utilities/configuration/application.json";
-    private static final String KEYWORD_OVERRIDDEN_CONFIGURATION = "org/openstreetmap/atlas/utilities/configuration/keywordOverridingApplication.json";
-    private static final String KEYWORD_OVERRIDDEN_DEV_CONFIGURATION = "org/openstreetmap/atlas/utilities/configuration/developmentOverriding.json";
-    private static final String OVERRIDE_CONFIGURATION = "org/openstreetmap/atlas/utilities/configuration/development.json";
-    private static final String PARTIAL_CONFIGURATION = "org/openstreetmap/atlas/utilities/configuration/feature.json";
+    private static final String KEYWORD_OVERRIDDEN_CONFIGURATION_JSON = "org/openstreetmap/atlas/utilities/configuration/keywordOverridingApplication.json";
+    private static final String KEYWORD_OVERRIDDEN_CONFIGURATION_YAML = "org/openstreetmap/atlas/utilities/configuration/keywordOverridingApplication.yml";
+
+    private static final String KEYWORD_OVERRIDDEN_DEV_CONFIGURATION_JSON = "org/openstreetmap/atlas/utilities/configuration/developmentOverriding.json";
+    private static final String KEYWORD_OVERRIDDEN_DEV_CONFIGURATION_YAML = "org/openstreetmap/atlas/utilities/configuration/developmentOverriding.yml";
+
+    private static final String OVERRIDE_CONFIGURATION_JSON = "org/openstreetmap/atlas/utilities/configuration/development.json";
```

Comment:
Would it be nicer to use `MergedConfigurationTest.class.loadResourceAsStream` instead below, so the package paths do not have to be hardcoded here? It would help in any future refactor.